pkg/handlers/snapshots.go

```diff
@@ -431,6 +444,11 @@ func configureMinioFileSystemProvider(ctx context.Context, clientset kubernetes.
 		ForceReset:       fileSystemOptions.ForceReset,
 		FileSystemConfig: fileSystemOptions.FileSystemConfig,
 	}
+
+	if _, err := os.Stat(*deployOptions.FileSystemConfig.HostPath); os.IsNotExist(err) {
```

Comment:
would be a good idea to catch other errors as well, something like
```go
if _, err := os.Stat(*deployOptions.FileSystemConfig.HostPath); os.IsNotExist(err) {
    return &kotssnapshot.HostPathFileSystemError{Message: "provided host path does not exist"}
} else if err != nil {
    return errors.Wrap(err, "failed to os stat")
}
```

Reply:
ah ofcourse, addressed